mgr/Conscript-standard

```diff
@@ -11,12 +11,8 @@ my $STAR_SYS     = $env->{ENV}->{STAR_SYS};
 my $STAR_VERSION = $env->{ENV}->{STAR_VERSION};         # print "#####  $STAR_VER\n";
 my $AFS_RHIC     = $env->{ENV}->{AFS_RHIC};
 
-my $ROOT_FEATURES = `root-config --features`;
```

Comment:
What about the case when root is built with the tables? Are include/library paths preferring our version?

Reply:
Yes, in general everything in our build/install directory (i.e. .slXX_gccNNN) precedes the external locations. With those conditions in Conscript we have never really built or tested the local TTable because both ROOT 5.34 and 6.16 still have libTable support.

There is, however, a different issue... Our local TTable in StRoot is from ROOT 6.16 and it is not compatible with ROOT 5.34. The CI fails due to that. Should we freeze our TTable at ROOT 5.34 if it still works with ROOT6 as dependency?

Reply:
The error states that our TTable provides 8 arguments to `TDefaultInitBehavior::CreateClass` instead of 9. I doubt we can resolve this by just downgrading TTable. Either would have to have conditionals, or give up on providing our TTable for ROOT5.

Reply:
Just for the record, I went with the second option, i.e. our local TTable is built and used only when ROOT version is 6 and higher. The case of ROOT5 should not be affected by this. The PR's description has been updated accordingly.